# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/AudioStream.cpp
# Commit: 150ecd10e53a
# Full Hash: 150ecd10e53a7e51b563bc8595638516e969aa3b
# Author: Paul Adenot <paul@paul.cx>
# Date: 2022-05-10 03:50:31
# Regressor Bug: 1743834
# File Overlap Count: 1
# Description:
#   Bug 1743834 - Don't start with an AudioSink if muted from the start. r=alwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D141990
# ==============================================================================

diff -r 4076e2055960 -r 150ecd10e53a dom/media/AudioStream.cpp
--- a/dom/media/AudioStream.cpp	Mon May 09 13:20:51 2022 +0000
+++ b/dom/media/AudioStream.cpp	Mon May 09 13:20:51 2022 +0000
@@ -149,8 +149,7 @@
       mPlaybackComplete(false),
       mPlaybackRate(1.0f),
       mPreservesPitch(true),
-      mCallbacksStarted(false)
-    {}
+      mCallbacksStarted(false) {}
 
 AudioStream::~AudioStream() {
   LOG("deleted, state %d", mState.load());
@@ -233,7 +232,8 @@
   auto startTime = TimeStamp::Now();
   TRACE("AudioStream::Init");
 
-  LOG("%s channels: %d, rate: %d", __FUNCTION__, mOutChannels, mAudioClock.GetInputRate());
+  LOG("%s channels: %d, rate: %d", __FUNCTION__, mOutChannels,
+      mAudioClock.GetInputRate());
 
   mSinkInfo = aSinkInfo;
 